ui/src/timeMachine/actions/queries.ts

```diff
@@ -115,6 +116,8 @@ export const executeQueries = () => async (dispatch, getState: GetState) => {
   try {
     dispatch(setQueryResults(RemoteDataState.Loading, [], null))
 
+    await dispatch(hydrateVariables())
```

Comment:
this refreshes variables that haven't been fetched by another means before sending off the query, just in case. found this while deep linking w/ dependent variables and no application state.